test/pointtileindex.cc

```diff
@@ -1,6 +1,9 @@
 #include "midgard/point_tile_index.h"
 #include "test.h"
 
+// If supporting C++20 it'd be better to use std::numbers::pi from <numbers>.
+constexpr double kPI = 3.14159265358979323846;
```

Comment:
msvc doesn't support `M_PI`?

Reply:
You need to define `_USE_MATH_DEFINES` before _any_ `math.h` or `cmath` includes. Not standard C++.

Since it must be before _any_ includes - `_USE_MATH_DEFINES` would have to be defined as a project define. Could do that as an alternative.

Reply:
Ah, didn't know that. Na, it's fine as-is, mid/short-term we do want to move to C++20. I can't exactly remember if there was some kind of blocker, probably not actually. We still don't use many of the 17 goodies, but that's not a blocker and will be handled along the way.